reference/resources/felixconfig.md

```diff
@@ -39,6 +39,7 @@ spec:
 
 | Field                              | Description                 | Accepted Values   | Schema | Default    |
 |------------------------------------|-----------------------------|-------------------|--------|------------|
+| awsSrcDstCheck                     | Controls automatically setting {% include open-new-window.html text='source-destination-check' url='https://docs.aws.amazon.com/vpc/latest/userguide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck' %} on an AWS EC2 instance running Felix. Setting the value to `Enable` will set the check value in the instance description to `true`. For `Disable`, the check value will be `false`. Setting must be `Disable` if the EC2 instance can process traffic not matching interface IP address. For example, EKS cluster using Calico CNI with `VXLANMode=CrossSubnet`. Check [IAM role and profile configuration](#aws-iam-rolepolicy-for-source-destination-check-configuration) for setting the necessary permission for this setting to work.| DoNothing, Enable, Disable | string | `DoNothing` |
```

Comment:
"if the EC2 instance can process" -> "if you want the EC2 instance to process"
"matching interface IP address" -> "matching the host interface IP address"